ddtrace/internal/schema/__init__.py

```diff
@@ -17,7 +17,8 @@ def _validate_schema(version):
         "the value exported in the 'DD_TRACE_SPAN_ATTRIBUTE_SCHEMA' environment variable.",
     )
 
-    assert version in _SPAN_ATTRIBUTE_TO_FUNCTION.keys(), error_message
+    if version not in _SPAN_ATTRIBUTE_TO_FUNCTION.keys():
```

Comment:
My approach in Node is to fallback to `v0` if the user provides garbage, and warn about it. It might be good to pick a solution and have the same across tracers.

Reply:
as per offline discussion, I will be updating the code to default to `v0` on invalid input